docs/fiddles/features/drag-and-drop/index.html

```diff
@@ -10,6 +10,7 @@ <h1>Hello World!</h1>
     We are using node <script>document.write(process.versions.node)</script>,
     Chrome <script>document.write(process.versions.chrome)</script>,
     and Electron <script>document.write(process.versions.electron)</script>.
+    <p></p>
```

Comment:
Would be more semantically correct to wrap the "We are using..." sentence with the `<p>` tags.